src/sentry/api/serializers/models/organization_member.py

```diff
@@ -34,3 +36,42 @@ def serialize(self, obj, attrs, user):
             'dateCreated': obj.date_added,
         }
         return d
+
+
+class OrganizationMemberWithTeamsSerializer(OrganizationMemberSerializer):
+    def get_attrs(self, item_list, user):
+        attrs = super(OrganizationMemberWithTeamsSerializer,
+                      self).get_attrs(item_list, user)
+
+        member_team_map = OrganizationMemberTeam.objects.values(
```

Comment:
i think you wanna 1) filter this to only include the members you're trying to serialize and 2) cast it to a list to ensure the query is only evaluated once even if you use it multiple times below. I think something like:
```python
member_team_map = list(OrganizationMemberTeam.objects.filter(
    organizationmember__in=item_list,
).values('organizationmember', 'team'))
```